### ISSUE TITLE
Create a page for Volto CI

### ISSUE THREAD

**Issue by stevepiercy** [plone/volto#5883]
And now that I think about it, most of the new-to-GitHub users have no idea of what is CI and how to use it. I think we should have either a new section or page specifically for CI, just as we have separate pages for each thing that a contributor needs to know. I think this is worth a separate issue and PR itself, so that we don't wander too far off course in this PR.

_Originally posted by @stevepiercy in https://github.com/plone/volto/pull/5882#discussion_r1527137050_

**Comment by sammyifelse** [comment]
Hey @stevepiercy  are you working on this issue

**Comment by shashank0470** [comment]
Can  i work on this ?

**Comment by stevepiercy** [comment]
@shashank0470 no. Please read and follow [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), especially [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do), [Contributing to Plone](https://6.docs.plone.org/contributing/index.html), and [Contributing to Volto](https://6.docs.plone.org/contributing/volto.html).
